nexus_constructor/treeview_utils.py

```diff
@@ -280,3 +286,5 @@ def get_module_frame(frame: QFrame, model: Model, value: FileWriterModule):
     module_frame = ModuleView(value, frame, model)
     frame.module_frame = module_frame
     frame.layout().addWidget(frame.module_frame)
+    if isinstance(value, Dataset) and value.name in EXCLUDED_PIXEL_GRID:
```

Comment:
I would suggest to not show at all. Disabling it doesn't help too much as one can not see the content because even Edit button will be disabled and array table won't be visible.